packages/envd/internal/api/init.go

```diff
@@ -52,15 +52,6 @@ func (a *API) PostInit(w http.ResponseWriter, r *http.Request) {
 
 	logger.Debug().Msg("Syncing host")
 
```

Comment:
```suggestion
```

Reply:
There is sync of metadata coming from Firecracker, so it's still relevant IMHO.

Reply:
but that isn't syncing host, is it?